Project.toml

```diff
@@ -12,8 +12,8 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 
 [compat]
 DataFrames = "≥ 0.17.1"
-LinearOperators = "≥ 0.4.0"
-NLPModels = "0.7.0, 0.8.0, 0.9"
+LinearOperators = "0.4.0, 0.5.0, 0.6.0, 0.7.0"
```

Comment:
Why list the versions instead of specifying a range?

Reply:
Looks like it doesn't work yet: https://discourse.julialang.org/t/explicit-version-ranges-in-compat-section-in-project-toml/30147

Reply:
OK.